plugin.py

```diff
@@ -46,5 +47,56 @@ def on_pre_server_command(self, params: Mapping[str, Any], done_callback: Callab
             done_callback()
         return True
 
+    def on_server_response_async(self, method: str, response: Response) -> None:
+        if method == 'codeAction/resolve':
+            self.maybe_handle_move_code_action(response)
 
+    def maybe_handle_move_code_action(self, response: Response) -> None:
+        session = self.weaksession()
+        if not session:
+            return
+        result = cast(MaybeMoveFunctionCodeAction, response.result)
+        command = result.get('command')
+        if not command:
+            return
+        command_name = command.get('arguments')[0]
+        params = command.get('arguments')[1]
+        function_name = command.get('arguments')[2]
+        if command_name == 'moveFunction':
+            move_params = {
+                'sourceUri': params.get('textDocument').get('uri'),
+                'params': params
+            }  # type: MoveParams
+            get_move_destinations_request = Request('elm/getMoveDestinations', move_params)
+            session.send_request(get_move_destinations_request, lambda res: self.on_get_destinations(res, params, function_name))
 
+    def on_get_destinations(self, response: MoveDestinationsResponse, params: MoveParamsParams, function_name: str) -> None:
+        destinations = response.get('destinations')
+        if not destinations:
+            sublime.status_message('LSP-elm: No destinations to choose.')
+        session = self.weaksession()
+        if not session:
+            return
+        window = sublime.active_window()
+        if not window:
+            return
+
+        def on_done(index) -> None:
+            if index == -1:
+                return
+            destination = destinations[index]
+            move_params = {
+                'sourceUri': params.get('textDocument').get('uri'),
+                'params': params,
+                'destination': destination
+            }  # type: MoveParams
+            move_request = Request("elm/move", move_params)
+            session.send_request(move_request, self.on_move_function)
```

Comment:
can probably be this to avoid empty method definition

```suggestion
            session.send_request(move_request, lambda any: pass)
```